src/components/policy/policy_regular/src/policy_manager_impl.cc

```diff
@@ -1118,8 +1120,9 @@ void PolicyManagerImpl::set_cache_manager(
 
 void PolicyManagerImpl::RetrySequence() {
   LOG4CXX_INFO(logger_, "Start new retry sequence");
-  RequestPTUpdate();
 
+  BinaryMessageSptr pt_snapshot = RequestPTUpdate();
```

Comment:
@LitvinenkoIra 
As PROPRIETARY mode includes **encryption** of snapshot by HMI and you can't send just plain table content w/o encryption. So you have to use previous snapshot received from HMI.